src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -124,6 +124,11 @@ import {
 import gridWidths from './gridWidths';
 
 import { BrandPalette, Typography, BrandSVG } from '../../models/types/theming';
+import { PageTypes } from '../../models/types/global';
+
+const isDarkUiPage = (pageType: PageTypes, derivedPageType?: string | null) =>
+  pageType === 'mediaArticle' ||
+  (pageType === 'media' && derivedPageType?.toLowerCase() === 'on demand tv');
```

Comment:
We need  an additional check because OnDemand Audio and OnDemand TV share the same `pageType` value of `media`.

We only want to style OnDemand TV in the dark UI.